charts/rancher-cis-benchmark/6.5.0/templates/validate-install-crd.yaml

```diff
@@ -0,0 +1,17 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
```

Comment:
Similar to the other PR, this file no longer exists in the [source of truth](https://github.com/rancher/cis-operator/tree/release/v1.2/chart/templates).

Reply:
added this file but it is being deleted automatically when we run make charts